pkg/scanners/scannerv4/scannerv4.go

```diff
@@ -319,6 +324,32 @@ func (s *scannerv4) TestNodeScanner() error {
 	return nil
 }
 
+func (s *scannerv4) GetVirtualMachineScan(
+	vm *storage.VirtualMachine,
+	indexReport *v4.IndexReport,
+) (*storage.VirtualMachineScan, error) {
+	if s.scannerClient == nil {
+		return nil, errors.New("Scanner V4 client not available for VM enrichment")
+	}
```

Comment:
super nit: this does not appear needed - the scanner will never be initialized if creating the scannerClient fails.